.github/workflows/publish_imgs.yml

```diff
@@ -70,18 +70,33 @@ jobs:
     # Utilizing nixos/nix docker image v2.3
     container:
       image: nixos/nix@sha256:af330838e838cedea2355e7ca267280fc9dd68615888f4e20972ec51beb101d8
-    
+
     steps:
       # Checks out repository to docker container
       # Utilizes the commit from the open PR, which was gotten from the previous step
       # Using SHA v2
-      - name: Pulls Repository Code
+      - name: Pulls repository code
         uses: actions/checkout@28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b
         with:
           ref: ${{ needs.verify.outputs.sha }}
 
+      - name: Install bash
+        run: |
+          mkdir /github/home/.nix-defexpr
+          ln -s /nix/var/nix/profiles/default /github/home/.nix-profile
+          ln -s /nix/var/nix/profiles/per-user/root/channels /github/home/.nix-defexpr/channels
+          nix-env -iA nixpkgs.bash
+        shell: sh {0}
+        continue-on-error: false
+
+      - name: Install Cachix
+        uses: cachix/cachix-action@73e75d1a0cd4330597a571e8f9dedb41faa2fc4e  # v10
+        with:
+          name: nebulaworks
+          authToken: ${{ secrets.CACHIX_AUTH_TOKEN }}
```

Comment:
@sarcasticadmin FYI we'll need to add this value to the main `nix-garage` repo. It's in 1pass.

Reply:
Thanks for the reminder, its been added